config/auto_battle_operation/般岳-连招-怒相连招.sample.yml

```
@@ -0,0 +1,54 @@
+# 般岳-连招-怒相连招
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)